src/reference/asciidoc/debezium.adoc

```diff
@@ -160,7 +164,7 @@ public void handler(List<ChangeEvent<Object, Object>> payload) {
 ----
 ====
 
-=== [[configuring-with-the-java-dsl]]Configuring with the Java DSL
+=== [[debezium-java-dsl]]Debezium Java DSL Support
```

Comment:
Something wrong with this code flow.
Must be like this:
```
[[debezium-java-dsl]]
=== Debezium Java DSL Support
```

Reply:
OK. Good to hear that it works, but it is not readable with that style.
Probably the anchor `[[]]` works from any point of text, even in the middle of the sentence.
but does it really make sense to do that?